doc/spec/pantry-spec.md

```diff
@@ -0,0 +1,66 @@
+# Roux Pantry Spec
+
+Ingredients in the Roux ecosystem are distributed in a **pantry**: a filesystem
+subtree containing one or more ingredients.
+
+For example:
+
+```
+┌── my-project
+│   └── pantry
+│       ├── forms
+│       │   └── index.scss
+│       │   └── index.hbs
+│       │   └── ingredient.md
+│       ├── sidebar
+│       │   └── index.scss
+│       │   └── index.hbs
+│       │   └── ingredient.md
+│       │   └── model.js
+```
+
+Pantries are typically named `pantry` but can be named anything you choose.
+A consuming application will define the `pantryRoot` in its `package.json`:
+
+```
+ "roux": {
+   "pantryRoot": "pantry"
+ }
+```
+
+
+## `pantry.js`
+
+A pantry is defined by the presence of a file named `pantry.js` in the root
+level pantry directory (above, `pantry`). A pantry cannot contain another pantry.
```

Comment:
✅  revised the first bit. haven't yet made any clarifications about pantry.js vs pantryRoot in package.json.